tests/unit/components/gh-selectize-test.js

```diff
@@ -31,8 +31,8 @@ describeComponent(
                 component._selectize.setValue(['item 3', 'item 2']);
             });
 
-            expect(component.get('value'), 'component value').to.deep.equal(['item 3', 'item 2']);
-            expect(component.get('selection'), 'component selection').to.deep.equal(['item 3', 'item 2']);
+            expect(component.get('value').toArray(), 'component value').to.deep.equal(['item 3', 'item 2']);
```

Comment:
For some reason, Safari handles ember arrays differently than Chrome/Firefox. To get around that, we cast the ember array to a regular array to verify the deep equal. (Note: I tried wrapping the expected value in an emberA - which would look like `expect(someEmberArray).to.deep.equal(emberA(['a', 'b']));` but that didn't work either)